lib/l10n/app_en.arb

```diff
@@ -1,6 +1,24 @@
 {
   "@@locale": "en",
 
+  "price": "Preis: {amount}€",
+  "@price": {
+      "placeholders": {
+          "amount": {
+              "example": "123.45",
+              "description": "amount"
+          }
+      }
+  },
+  "cancel": "cancel",
+  "confirm": "confirm",
+  "seat": "{count, plural, =1{seat} other{seats}}",
```

Comment:
this should be called "seats" because it's the pluralization

Reply:
Actually, on further inspection, is that even used anywhere?

Reply:
not right now but we'll need it later for the detail views. I first used it but then i changed it i can take it out if you want

Reply:
Yeah maybe just take it out, I'll do it

As a sidenote, the string should contain the number itself probably